zulipterminal/model.py

```diff
@@ -1,9 +1,9 @@
-import datetime
 import json
 import time
 from collections import OrderedDict, defaultdict
 from concurrent.futures import Future, ThreadPoolExecutor, wait
 from copy import deepcopy
+from datetime import datetime
```

Comment:
:+1: I made some changes here but is the part where I mention "this is a prep commit" fine?

Reply:
The problem with refactoring `test_ui_tools` is that `pytest` and `mypy` would get confused between the mocked `date` and the actual `date`. So its better to keep it as `datetime.date`

Reply:
I don't think it should be not an issue if we get the naming correct.

Reply:
It's just that we should be consistent with the imports or other conventions throughout the source. :)

Reply:
@preetmishra I was getting errors as it was using the patched `date` as the imported `date`.
I think you meant "don't think it should be an issue" or otherwise?

Reply:
You could change the variable name to differentiate between the two.

Yeah, that's what I meant. :) (I have amended my comment.)